@xen-orchestra/lite/src/stories/web-core/state-hero/vts-object-not-found-hero.story.vue

```diff
@@ -1,5 +1,11 @@
 <template>
-  <ComponentStory v-slot="{ properties }" :params="[prop('id').required().str().preset('123-456-789').widget()]">
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('id').required().str().preset('123-456-789').widget(),
+      prop('type').required().enum('page', 'card', 'table', 'panel').preset('card').widget(),
```

Comment:
Since the prop is added here and used in the `[id].vue` components, it should be added in `VtsObjectNotFoundHero.vue` component as well. It might've been deleted in the last rebase. I remember you added it.